Automattic/wp-calypso
# composite-checkout: Add existing card payment method

#### Changes proposed in this Pull Request

This PR adds payment methods for existing credit cards to composite checkout.

When the checkout form in Calypso is first loaded, it fetches the existing cards from the REST API and then creates a new Payment Method Object for each one using the new `createExistingCardMethod()` function that's part of composite-checkout's public API.

The payment method label and inactive content are not currently styled, but that can come in a later PR.

#### Testing instructions

- Sandbox the store.
- Using calypso master, add a saved credit card to your account. It can be a Stripe test card, eg: `4242 4242 4242 4242`.
- Run calypso with `ENABLE_FEATURES=composite-checkout-wpcom npm start`.
- Add a plan to your cart and visit checkout.
- Verify that you see a payment method in the list for your existing card.
- Select the existing card and complete the payment form.
- Verify that the payment is successful and that the plan is added to your site successfully.
